### PR TITLE
Fixed enum syntax error. Should use [] instead of {}

### PR DESCRIPTION
**Pull request by hughesbm**
Here's the relevant error log from heroku:

screen shot 2016-10-02 at 12 54 39 pm

### CODE REVIEW

File: app/models/game.rb [hunk 1]
```diff
@@ -2,5 +2,5 @@ class Game < ActiveRecord::Base
   has_many :pieces
   belongs_to :white_player, class_name: 'User'
   belongs_to :black_player, class_name: 'User'
-  enum state: { in_progress: 0, complete: 1 }
+  enum state: [in_progress: 0, complete: 1]
```
Comment (ijones16) [inline app/models/game.rb]: When using an array, the order is implied I believe

http://edgeapi.rubyonrails.org/classes/ActiveRecord/Enum.html

Comment (hughesbm) [inline app/models/game.rb]: Thanks for the link. Looks like the hash syntax is valid; I wonder why heroku was giving an error?

**Comment by hughesbm** [comment]
Looking at the error logs again-- it's looking in app/vendor/bundle/ruby/2.2.0. I think we're supposed to be on 2.3.1. Is that what's causing the heroku crashes?

**Comment by hughesbm** [comment]
I looked at the most recent heroku build log and found this:
```
###### WARNING:
       You have not declared a Ruby version in your Gemfile.
       To set your Ruby version add this line to your Gemfile:
       ruby '2.2.4'
       # See https://devcenter.heroku.com/articles/ruby-versions for more information.
```
Which is weird because we do have ruby '2.3.1' in our Gemfile. I followed the link and went through the troubleshooting section, but it doesn't seem like any of those cases apply here. I'm stumped.

**Comment by hughesbm** [comment]
Wow. I just realized rubocop already fixed this problem in the master branch, it just hasn't been deployed to heroku yet.

iwkad22 1

**Pull request closed without merging**